backend/src/database/entities/personnel.entity.ts

```diff
@@ -90,8 +92,14 @@ export class PersonnelEntity extends BaseEntity {
   @Column({ name: 'application_date', type: 'timestamp' })
   applicationDate: Date;
 
-  @Column({ name: 'supervisor', type: 'varchar', length: 100 })
-  supervisor: string;
+  @Column({ name: 'supervisor_first_name', type: 'varchar', length: 100 })
+  supervisorFirstName: string;
+
+  @Column({ name: 'supervisor_last_name', type: 'varchar', length: 100 })
+  supervisorLastName: string;
+
+  @Column({ name: 'supervisor_email', type: 'varchar', length: 50 })
```

Comment:
as per your comment in gov channel this morning, should we make this `nullable: true`?